src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,28 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.FormatStyle;
+
+public class Deadline extends Task {
+    protected LocalDate by;
```

Comment:
Might want to consider changing the variable name to 'deadline' instead because it wasn't very obvious what this variable was referring to at first glance.